### PR TITLE
Merge eckoh payments into master

### PR DESCRIPTION
**Pull request by jtbainbridge**
Created individual pages for the payment screens. These will need to be exported into separate plain html/css files post review/QA

### CODE REVIEW

File: src/components/payments/_global/PaymentStatusError.vue [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        <a class="dcww-payments__link dcww-payments__link--image" href="https://www.dwrcymru.com/"><img src="../../../assets/dcww.png" alt="logo" class="dcww-logo"></a>
+        <div class="dcww-payments-header__links">
+            <a class="dcww-payments__link" href="">Cymraeg</a>
+        </div>
+    </header>
+
+    <main class="dcww-payments-form">
+        <div class="dcww-payments-form__container">
+            <div class="dcww-payments-status">
+                <img src="../../../assets/caution.svg" class="dcww-payments-status__icon--large">
```
Comment (lam218) [inline src/components/payments/_global/PaymentStatusError.vue]: all of these svgs I would consider doing inline as eckoh are just going to have the plain html right? None of the other files

File: src/components/payments/guest/GuestStep1.vue [hunk 2]
```diff
@@ -0,0 +1,494 @@
...
+    color: white;
+}
+
+.dcww-legal-copy__copyright {
+    color: white;
+}
+
+.dcww-fieldset {
+    /* display: flex;
+    flex-flow: column; */
```
Comment (lam218) [inline src/components/payments/guest/GuestStep1.vue]: remove commented code

File: src/components/payments/guest/GuestStep1.vue [hunk 3]
```diff
@@ -0,0 +1,494 @@
...
+    align-items: center;
+    box-sizing: border-box;
+    border-radius: 4px;
+    display: inline-flex;
+    line-height: 1.75;
+    font-size: 18;
+    font-family: 'Nunito', sans-serif;
+    font-weight: bold;
+    padding: 9px 20px;
+    transition: box-shadow 0.25s ease-in-out, background 0.25s ease-in-out, color 0.25s ease-in-out, border 0.25s ease-in-out;
```
Comment (lam218) [inline src/components/payments/guest/GuestStep1.vue]: if they are all having the same transition, you can just do ```transition: all 0.25s ease-in-out;```

File: src/components/payments/guest/GuestStep1.vue [hunk 4]
```diff
@@ -0,0 +1,494 @@
...
+
+.dcww-button--secondary:hover {
+    -webkit-box-shadow: 0 5px 10px 0 rgba(0,0,0,.15);
+    box-shadow: 0 5px 10px 0 rgba(0,0,0,.15);
+    color: #009edb;
+}
+
+.dcww-button--chevron {
+    padding-right: 45px;
+    background-image: url('../../../assets/chevron-right.svg');
```
Comment (lam218) [inline src/components/payments/guest/GuestStep1.vue]: same about inline earlier

File: src/components/payments/registered/RegisteredAppFee.vue [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+    min-height: 60px;
+    margin-left: 10px;
+    padding: 18px 35px 18px 20px;
+    width: 100%;
+    font-family: 'Open Sans', sans-serif;
+    font-size: 1rem;
+    font-weight: 300;
+    color: #09333f;
+    line-height: normal;
+}
```
Comment (lam218) [inline src/components/payments/registered/RegisteredAppFee.vue]: does this dropdown stuff need repeating?

File: src/components/payments/_global/PaymentStatusError.vue (hunk 1 above)
Comment (jtbainbridge) [reply src/components/payments/_global/PaymentStatusError.vue]: We are sending them an assets bundle

File: src/components/payments/guest/GuestStep1.vue (hunk 4 above)
Comment (jtbainbridge) [reply src/components/payments/guest/GuestStep1.vue]: as above

File: src/components/payments/registered/RegisteredAppFee.vue (hunk 5 above)
Comment (jtbainbridge) [reply src/components/payments/registered/RegisteredAppFee.vue]: Good catch - removed

**Pull request merged**